### DISCUSSION TITLE
A few suggestions regarding the new playlist layout

### DISCUSSION THREAD

**Discussion by alkapwn3d** [FreeTubeApp/FreeTube#4884] [General]
```
OS: Arch Linux
Architecture    : x86_64
Installer: paru

Name: freetube
Version: 0.20.0-1
Build Date      : Thu 04 Apr 2024 06:29:07 PM EDT
Install Date    : Fri 05 Apr 2024 02:41:37 PM EDT
```

Just wanted to give some feedback on the recent changes. 

Someone already reported it but theres a new bug after the recent update that makes the video stop playing when ever I switch to a different workspace. 

I preferred the previous playlist view that let me see multiple rows and columns sorted by most recently added items. Its one feature I liked that youtube doesnt have.

Not really a fan of the new playlist view thats sorted oldest to newest in a single column. I have over 1000 items in my favorites and i have to scroll all the way down. and it doesnt remember the state so if i leave and come back i have to do it all again. Its just more of a hassle to navigate through all my videos. maybe you can have an option to select or toggle the old view

Also id prefer if I had a separate "quick bookmark" button to add to watch list rather than just to favorites. 

Also id like to be able to just add an existing playlist to my own playlist collection, without having to create a new playlist then adding copying it. seems like a lot of extra steps

great app btw. 
keep up the good work

**Comment by jnihil** [comment]
I'm very grateful for the effort that has gone into extending the playlist feature to cater for multiple playlists, but I totally agree with the suggestions from @alkapwn3d above.

**Comment by absidue** [comment]
> Also id prefer if I had a separate "quick bookmark" button to add to watch list rather than just to favorites.

You can choose which playlist you want to be the quick bookmark one, by clicking the link icon inside the playlist.

**Comment by xkix** [comment]
Great App, thanx!
I' am also having > 1000 items, new playlist layout is unusable for me, have to downgrade unfortunately.

**Reply by xkix** [reply]
Thanks you so much for fixing this issue!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
We are aware of the improvements we can do for FT and if you attemped a search on our issue tracker you can see that for yourself. 

This is just the initial release for the Playlists feature after almost a year of development. 

More improvements will come. 

> Not really a fan of the new playlist view thats sorted oldest to newest in a single column

Documented in #4870

> and it doesnt remember the state so if i leave and come back i have to do it all again

Documented in #2446

> Also id prefer if I had a separate "quick bookmark" button to add to watch list rather than just to favorites.

As @absidue said this is already implemented

> Also id like to be able to just add an existing playlist to my own playlist collection, without having to create a new playlist then adding copying it. seems like a lot of extra steps

You dont have to create a new playlist if you want to copy a playlists into an existing one, try it.
